### DISCUSSION TITLE
Duplicated script tag when deploying Nuxt3 app

### DISCUSSION THREAD

**Discussion by ahmedsalhin** [vercel/community#2886] [Help] [answered]
### Summary

I am trying to add cookies model to my Nuxt3 app using [cookieyes](https://www.cookieyes.com/documentation/getting-started/installation-guides/cookie-banner-on-nuxt-js/). 

Nuxt3 config appears as:

```js
export default defineNuxtConfig({
  app: {
    head: {
      script: [
        {
          src: "https://cdn-cookieyes.com/client_data/TOKEN/script.js",
          id: "cookieyes",
          type: "text/javascript"
        }]}
  },
```
Generating the distribution locally shows index.html with the correct script tag. 

Deploying to Vercel also shows the following in the output code

```html
head><meta charset="utf-8">
<title>XYZ</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<script src="https://cdn-cookieyes.com/client_data/TOKEN/script.js" id="cookieyes" type="text/javascript"></script>
<style>body { overflow: overlay }</style>
....
```

However, when I inspect the website on the browser, I can see another script added like here 

```html
head><meta charset="utf-8">
<title>XYZ</title>
<meta name="viewport" content="width=device-width, initial-scale=1">
<script src="https://cdn-cookieyes.com/client_data/TOKEN/script.js" id="cookieyes" type="text/javascript"></script>
<script src="https://cdn-cookieyes.com/client_data/TOKEN/banner.js" async="" id="cookieyes-banner"></script>
....
```

This causes cookieyes not to activate the cookie banner, as I suspect it sees two script with id starts with `cookieyes`

### Steps to Reproduce

Vercel deploy Nuxt3 app

nuxt.config.ts

```js
export default defineNuxtConfig({
  app: {
    head: {
      script: [
        {
          src: "https://cdn-cookieyes.com/client_data/TOKEN/script.js",
          id: "cookieyes",
          type: "text/javascript"
        }]}
  },
```

**Comment by amyegan** [comment, marked as answer]
This extra script tag doesn't seem to be added by Vercel. I think `banner.js` is being added by the initial CookieYes script to let it show the cookie banner. Do you see any warnings or errors in the dev tools console? Or any errors in the network tab?

The CookieYes site has a nice [troubleshooting guide](https://www.cookieyes.com/troubleshooting-guide/). If that doesn't help you get it working, they also have a [support team](https://www.cookieyes.com/support/#troubleshoot) that can give you a bit more assistance with their tools.
